swan-lake/development-tutorials/source-code-dependencies/style-guide/annotations-documentation-and-comments.md

```diff
@@ -106,8 +80,8 @@ function func1() {
 }
 
 function func2() {
-    if (true) {
-        if (true) {
+    if isValid {
+        if isActive {
```

Comment:
```suggestion
    if valid {
        if active {
```

Functions use the `is` (`has`, etc.) prefix, right? Variables can be without `is`.

Reply:
These have to be updated wherever else applicable too.

Reply:
Fix the other occurrences also.